controlplane/control/rule.go

```diff
@@ -12,6 +13,15 @@ const (
 	SrlCel = iota + 1
 )
 
+func NewRuleLangFromString(s string) RuleLang {
+	switch s {
+	case "CEL":
+		return SrlCel
+	default:
+		return 0
```

Comment:
We should define and unknown type if we are going to use it

Reply:
Done